expr/component_test.go

```diff
@@ -0,0 +1,70 @@
+package expr
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestComponentEvalName(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name, want string
+	}{
+		{name: "", want: "unnamed component"},
+		{name: "foo", want: `component "foo"`},
+	}
+	for _, tt := range tests {
+		tt := tt
```

Comment:
Trying to understand why this is needed, I don't see the code below keeping a reference to the loop variable so it should be OK without this line?

Reply:
I wrote the code according to [Go wiki](https://github.com/golang/go/wiki/TableDrivenTests#parallel-testing) but it might be not always necessary.

Reply:
Oh looks like it is needed, thank you for the pointer, I learned something!